tools/webpacks/README.md

```diff
@@ -0,0 +1,17 @@
+Highcharts Webpacks
+===================
+
+Collection of configurations and plugins to create Highcharts bundles.
+See [Webpack documentation](https://webpack.js.org/concepts/) for concept and configuration details.
+
+
+
+Overview
+--------
+
+- `externals.json`: Maps imports, that should not be included, to the namespace. 
+  Only masters that are listet in the "included" option will bundle the listed imports.
```

Comment:
"Listet". Also might be good with an explanation of the `namespacePath` keyword. (Like what is the leading dot relative to, and should it be unique? etc)